dagmc_stats/DagmcStats.py

```diff
@@ -90,3 +92,23 @@ def __set_dagmc_tags(self):
                                                                      tag_type=tag_data['type'],
                                                                      storage_type=types.MB_TAG_SPARSE,
                                                                      create_if_missing=False)
+
+    def __set_entityset_ranges(self):
+        """
+        Set a dictionary with MOAB Ranges that are specific to the
+        types.MBENTITYSET type
+
+        inputs
+        ------
+        none
+
+        outputs
+        -------
+        none
+        """
+
+        entityset_types = ['Nodes', 'Curves', 'Surfaces', 'Volumes']
```

Comment:
I'd argue for this being either a Class variable or an object member variable (I still prefer the former for this and `self.entity_types`)